### ISSUE TITLE
Can't open TOTK-SaveGame-Editor.sln

### ISSUE THREAD

**Issue by psybovine** [lchmagKekse/TOTK-SaveGame-Editor#53]
Trying to open the app just starts "Visual Studio 2019". I've tried a few different things but even with .NET installed it's just not giving me any option to open the app.

**Comment by psybovine** [comment]
Nevermind, was confused on what I needed to download.

**Comment by shiipou** [comment]
Yeah openning the .sln with Visual Studio 2019 is for contributing, if you want to modify some part of the code or add more functionnality.
If you just want to use it, you just have to open the Release page and download the .exe

If everything is goot for you, you can close this issue. (That tell the dev this issue is fixed)
